<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingMenu</class>
 <widget class="QWidget" name="SettingMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="autoStartSet">
          <property name="text">
           <string>开机自启动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isAinmationSet">
          <property name="text">
           <string>动画效果</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>动画效果设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>移动设置</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>移动时长(毫秒)：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>移动距离(像素)：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>停留时间(秒)：</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QSpinBox" name="speedWidget">
                   <property name="maximum">
                    <number>999999999</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="distanceWidget">
                   <property name="maximum">
                    <number>999999999</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="intervalWidget">
                   <property name="maximum">
                    <number>99999999</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>时间设置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>秒：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>分：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>时：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>周：</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLineEdit" name="secondWidget"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="minuteWidget"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="hourWidget"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="dayWidget"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>isAinmationSet</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
